src/EA.Weee.Web.Tests.Unit/Controllers/OrganisationRegistrationControllerTests.cs

```diff
@@ -1455,5 +1455,32 @@ public async Task RepresentingCompanyRedirect_ShouldRedirectToType_WhenTransacti
             result.RouteValues["action"].Should().Be("Type");
             result.RouteValues["controller"].Should().Be("OrganisationRegistration");
         }
+
+        [Fact]
+        public void RegistrationComplete_Get_ReturnsView()
+        {
+            // Arrange
```

Comment:
@stevelee-civica  dont worry about an Arrange if there is nothing to arrange :)

Reply:
removed